### ISSUE TITLE
wandb.Video usage with np.uint8

### ISSUE THREAD

**Issue by bradyz** [wandb/client#634]
`wandb --version && python --version && uname`

* Weights and Biases version: 0.8.13
* Python version: 3.7.4
* Operating System: Linux

### Description

I'm trying to use wandb's Video object with np.uint8 -
should this following example work properly?

Because I couldn't get it to work without deleting these two lines in the wandb data_types file.
[https://github.com/wandb/client/blob/master/wandb/data_types.py#L624](https://github.com/wandb/client/blob/master/wandb/data_types.py#L624
)
### What I Did

minimal example of a gif that goes from blue to red

```
import numpy as np
import wandb


wandb.init()

video = list()

for a in np.arange(0, 1, 0.01):
    image = np.zeros((3, 64, 64), dtype=np.uint8)
    image[0] = a * 255
    image[2] = (1-a) * 255

    video.append(image)

wandb.log({'video': [wandb.Video(np.uint8(video), fps=50)]})
```

**Comment by lukas** [comment]
Thanks for flagging this.  It's a bug.  Should be fixed by https://github.com/wandb/client/pull/639.

**Comment by vanpelt** [comment]
Just released 0.8.14 with the fix.
